packages/bot-skeleton/src/scratch/utils/index.js

```diff
@@ -303,35 +350,49 @@ export const addDomAsBlock = (el_block, parent_block = null) => {
     return block;
 };
 
-export const hasAllRequiredBlocks = workspace => {
-    const blocks_in_workspace = workspace.getAllBlocks();
+export const isAllRequiredBlocksEnabled = workspace => {
     const trade_type_block = workspace.getAllBlocks(true).find(block => block.type === 'trade_definition_tradetype');
-    if (!trade_type_block) return false;
     const selected_trade_type = trade_type_block.getFieldValue('TRADETYPE_LIST');
     const mandatory_tradeoptions_block =
         selected_trade_type === 'multiplier' ? 'trade_definition_multiplier' : 'trade_definition_tradeoptions';
     const { mandatoryMainBlocks } = config;
     const required_block_types = [mandatory_tradeoptions_block, ...mandatoryMainBlocks];
-    const all_block_types = blocks_in_workspace.map(block => block.type);
-    const has_all_required_blocks = required_block_types.every(block_type => all_block_types.includes(block_type));
 
-    return has_all_required_blocks;
-};
+    const mandatory_blocks = ['trade_parameters', 'purchase_conditions'];
```

Comment:
Same comment as above "Can we put this in any of the config files where we are maintaining constant values? May be we can reuse this later?"

Reply:
done ✅